ledger-service/metrics/src/main/scala/com/digitalasset/http/metrics/HttpJsonApiMetrics.scala

```diff
@@ -27,6 +27,9 @@ class HttpJsonApiMetrics(
 ) {
   import HttpJsonApiMetrics._
 
+  @nowarn
+  def getDefaultMetricsFactory: MetricsFactory = defaultMetricsFactory
```

Comment:
I think better to expose just the `MetricRegistry` rather than the deprecated factory.

```
  @nowarn
  def getMetricRegistry : Option[MetricRegistry] = defaultMetricsFactory match {
    case mf: DropwizardMetricsFactory => Some(mf.registry)
    case _ => None
  } 
```

Reply:
done